.github/workflows/built-release.yml

```diff
@@ -11,4 +11,4 @@ jobs:
   built-release:
     uses: alleyinteractive/.github/.github/workflows/built-release.yml@main
     with:
-      node: 16
+      node: 20
```

Comment:
I'm guessing we can roll with defaults, since there's no JS in this plugin

Reply:
RE: your above comment about returns in docblocks, I'm only adding explanation when the return isn't obvious, ie in the case `mixed` primarily and when the return comes from a called function.  Keeping 20 LTS version here for now. Otherwise, I've made all of your other suggested changes, thank you! 🤘🏼